src/content/tutorials/observability/data-ingestion/logs/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Ingest logs into the Observability Platform
+linkTitle: Logs
+description: How to make new logs available in the Observability Platform in self-service.
+menu:
+  principal:
+    parent: tutorials-observability-data-ingestion
+    identifier: tutorials-observability-data-ingestion-logs
+weight: 50
+last_review_date: 2024-10-09
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - How to ingest new logs?
+  - Why do my clusters run Alloy?
+---
+
+By default, Giant Swarm clusters starting from release v29.0.0 are equipped with [Alloy](https://grafana.com/docs/alloy), it's a lightweight, high-performance log agent that collects system logs from your cluster and forwards them to a central [Loki](https://grafana.com/docs/loki) instance running on the management cluster.
+
+The Giant Swarm Observability Platform allows you to ingest logs from your workloads in a self-service way using labels to select which pods to ingest logs from.
+
+### Label your resources
+
+The label `giantswarm.io/logging` can be set to `true` or `false` on any Namespace or Pod. When the label isn't set it's considered `false`.
+
+- Pods in any namespace can be labeled
+- Namespace can be labeled, the value used for the namespace will be the default for all pods in this namespace
+- Pods label take precedence over Namespace label. For example logging can be enabled for a pod in a namespace where logging is disabled and vice versa.
+
+||Pod `giantswar.io/logging=true`|Pod `giantswar.io/logging=false`|Pod `unset`
+|-|------------------------------|--------------------------------|--------------------------
+**Namespace** `giantswar.io/logging=true`|<i class="fas fa-check"></i>|<i class="fas fa-times"></i>|<i class="fas fa-check"></i>
+**Namespace** `giantswar.io/logging=false`|<i class="fas fa-check"></i>|<i class="fas fa-times"></i>|<i class="fas fa-times"></i>
+**Namespace** `unset`|<i class="fas fa-check"></i>|<i class="fas fa-times"></i>|<i class="fas fa-times"></i>
+
+You can use the following command to label a namespace or a pod:
+
+```bash
```

Comment:
I really think we should push for gitOps on public documentation so the kubectl command IS not useful

Reply:
How can I achieve the same thing with git ops ?

Reply:
Customer label their namespace in git :)

Reply:
So it would be something like   
\> In the default way you can label a namespace in your git repository. Alternatively you can use the following command to label a namespace or a pod. 

or something like that, hm? I think this is still an intersting information how to do it alternatively.

Reply:
Added the mention about the git way.